display_list/display_list.h

```diff
@@ -74,19 +74,22 @@ namespace flutter {
                                     \
   V(SetBlender)                     \
   V(ClearBlender)                   \
-  V(SetShader)                      \
-  V(ClearShader)                    \
   V(SetImageFilter)                 \
   V(ClearImageFilter)               \
   V(SetPathEffect)                  \
   V(ClearPathEffect)                \
                                     \
   V(ClearColorFilter)               \
-  V(SetColorFilter)                 \
+  V(SetPodColorFilter)              \
```

Comment:
Curious, why "Pod"? Isn't that an implementation detail?

Reply:
Yes and no. Isn't "Clear vs Set" an implementation detail when the dispatcher only has a set method and takes null to mean "clear"?

The records aren't public, so basically this whole record mechanism is an implementation detail for Dispatcher.

The reason I went with Pod here is that many of the attribute records are only broken into ClearFoo and SetFoo where the SetFoo is capable of setting any sub-variant of Foo. In my case, some of the attributes can be inlined in the Pod space whereas others need to be explicitly held in the record itself so the compiler can properly dispose the dangling resource. This is true for all of the records I'm converting in these PRs because there is always an "Unknown" flavor of the attribute that simply holds an `sk_sp<SkSomething>` and we won't free that if it is in pod space. DlColorSource adds another one of those because it needs to have an explicitly declared `sk_sp<SkImage>` in the record. So it has the "DlUnknownColorSource" which needs to be broken out because of the `sk_sp<SkShader>` and also another breakout for `DlImageColorSource`. The ones that can just allocate/instantiate all of their data into the pod space are therefore special/similar in that they don't need an explicit member in the record. So, I could just call them SetFoo, but decided to make it clear that the records were holding a pod-allocated object.

And nobody really sees this except for the code in dl_ops.h and dl_builder.cc.

Reply:
Sounds good to me.